src/main/java/com/github/tomakehurst/wiremock/matching/MemoizingMatchResult.java

```diff
@@ -43,6 +43,11 @@ public Boolean get() {
   private final MatchResult target;
 
   public MemoizingMatchResult(MatchResult target) {
+    this(target, null);
+  }
+
+  public MemoizingMatchResult(MatchResult target, DiffDescription diffDescription) {
```

Comment:
This doesn't look like it should be here to me. This memoizing class is just a wrapper so the specifics of the match result should be entirely defined by the object being wrapped I think.